pkg/config/config.go

```diff
@@ -28,13 +28,15 @@ const (
 )
 
 type Network struct {
+	Name         string             `json:"name,omitempty"`
```

Comment:
Do we need to introduce the `Name` field? We use the network map key as the network name.
This will give users the chance to have a mismatch between the network key and the `Name` field.

Reply:
Removed it. Thank you.